compiler-cli/src/export.rs

```diff
@@ -28,9 +28,10 @@ pub(crate) fn erlang_shipment() -> Result<()> {
 
     // Build project in production mode
     let package = crate::build::main(Options {
-        perform_codegen: true,
         mode,
+        perform_codegen: true,
         target: Some(target),
+        warnings_as_errors: WarningLevel::Error,
```

Comment:
I don't think we should ever default to having this on as it means some people would not be able to deploy their code.